schema/person-record.json

```diff
@@ -157,12 +159,12 @@
   "$defs": {
     "Name": {
       "title": "Name",
-      "description": "A name by which this individual is known. Names should be provided in `fullName`, and may optionally be broken down in the `familyName`, `givenName` and `patronymicName` fields, based on the EC ISA Core Person Vocabulary (https://joinup.ec.europa.eu/solution/e-government-core-vocabularies) definitions.",
+      "description": "A name by which this individual is known. A name MUST be provided in `fullName`, and may optionally be broken down in the `familyName`, `givenName` and `patronymicName` fields, based on the EC ISA Core Person Vocabulary (https://joinup.ec.europa.eu/solution/e-government-core-vocabularies) definitions.",
       "type": "object",
       "properties": {
         "type": {
           "title": "Type",
-          "description": "What kind of name is this? See the nameType codelist.",
+          "description": "The status of this name for the individual, using the nameType codelist.",
```

Comment:
```suggestion
          "description": "The type of this name for this person, using the nameType codelist.",
```

Reply:
I've left as 'status' since it's a bit more meaningful